roles/openshift_etcd_server_certificates/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- include_role:
+    name: openshift_etcd_facts
```

Comment:
same as above.  If we are just using the variable locally, I'd argue we don't need an external call to set this fact on the host.  I'm not familiar with the `openshift_facts` code for `etcd` but I'd like to eliminate any calls to it if possible to reduce dependencies.

Reply:
Interesting, I can't find any place where the `openshift_etcd_server_certificates` role is used.